v1/proto/service/gribi.proto

```diff
@@ -257,6 +257,9 @@ message AFTResult {
   //
   // The timestamp is expressed as nanoseconds since the Unix Epoch in UTC.
   int64 timestamp = 3;
+
+  // Human-readable error message in the case that status is FAILED.
+  string error_message = 4;
```

Comment:
Wondering whether we should make this a message, if we ever want to have an `enum` of well known failure modes? e.g., one could envisage something like `FORWARD_REFERENCE` for where there are missing references?